applications/vessel-history/src/utils/vessel.test.tsx

```diff
@@ -15,7 +15,7 @@ describe('getVesselAPISource', () => {
     otherShipnames: [],
     shipname: 'DON TITO',
     source: 'AIS',
-    dataset: 'public-global-vessels:v20190502',
+    dataset: 'public-global-fishing-vessels:v20201001',
```

Comment:
Is this a unit test that mocks the API or you are actually calling the service?
If it is a mock you could change this to `vYYYYMMDD` so it is clear it is a mock.